Rubberduck.Inspections/Abstract/InspectionBase.cs

```diff
@@ -58,7 +58,7 @@ protected InspectionBase(RubberduckParserState state, CodeInspectionSeverity def
         /// <summary>
         /// Meta-information about why an inspection exists.
         /// </summary>
-        public virtual string Meta => InspectionsUI.ResourceManager.GetString(Name + "InspectionMeta", CultureInfo.CurrentUICulture);
+        public virtual string Meta => InspectionsUI.ResourceManager.GetString(Name + "Meta", CultureInfo.CurrentUICulture);
```

Comment:
The commit that introduced this is [this one](https://github.com/rubberduck-vba/Rubberduck/pull/3110/commits/6ab2b903f9238f566834f209d773f46efdec11ba)